docs/stations/Historical_Overseas_Stations.md

```diff
@@ -0,0 +1,30 @@
+## About overseas stations in Delta repository - Reference ##
+Historical long standing seismological international stations were refered to 
+in Delta/network/stations.csv and Delta/network/networks.csv as the "OE" network
+these have been removed since. 
+
+#### Station Recordings and Instrumentation ####
+Most likely used in the paper seismograms and pre-miniseed recordings. 
+	
+#### Countries Australia, U.S.A., Japan  ####
+Delta reference as it was for "historical stations": 
```

Comment:
The content here is ok, but if you preview this file it looks like github markdown isn't handling the line breaks/paragraphs here how you probably intended it to, and this csv block is all just running together instead of having each entry on its own line. Easiest fix would either be to surround this block in code tags (triple backtick \`\`\`) or add double spaces at the end of each line.